controllers/upgrade_handlers.go

```diff
@@ -230,8 +204,13 @@ func (r *ImageBasedUpgradeReconciler) postPivot(ctx context.Context, ibu *lcav1a
 		}
 		return requeueWithError(err)
 	}
+	if !result.IsZero() {
```

Comment:
What case is this handling? We're returning nil here, so I guess this means the restore didn't complete and we're expecting the reconciler to be called again? A comment here would be nice to have, as this isn't obvious (to me, at least)

Reply:
Exactly. Sure, will add comment in a further PR.